5.validate_model/scripts/nbconverted/cell_health_correlations.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### Import Libraries
+# 
+
+# In[1]:
+
+
+import sys
+import pathlib
+
+import pandas as pd
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt
+from ccc.coef import ccc
+from scipy.spatial.distance import squareform
+
+sys.path.append("../utils")
+import validate_utils
+
+
+# #### Set load/save paths
+# 
+
+# In[2]:
+
+
+# external path to be set
+classification_profiles_save_dir = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-plate-classification-profiles"
```

Comment:
I see your comment in the README about this - it's probably also worth adding some additional details in this notebook.

Are there plans to put the probabilities somewhere later?

Reply:
Added some additional details in this notebook. 

As discussed today, we plan on putting these valuable intermediate data on Zenodo in the near future.